doc/classes/GeometryInstance3D.xml

```diff
@@ -52,25 +52,27 @@
 		</member>
 		<member name="lod_bias" type="float" setter="set_lod_bias" getter="get_lod_bias" default="1.0">
 		</member>
-		<member name="lod_max_distance" type="float" setter="set_lod_max_distance" getter="get_lod_max_distance" default="0.0">
-			The GeometryInstance3D's max LOD distance.
-			[b]Note:[/b] This property currently has no effect.
+		<member name="material_override" type="Material" setter="set_material_override" getter="get_material_override">
+			The material override for the whole geometry.
+			If a material is assigned to this property, it will be used instead of any material set in any material slot of the mesh.
 		</member>
-		<member name="lod_max_hysteresis" type="float" setter="set_lod_max_hysteresis" getter="get_lod_max_hysteresis" default="0.0">
-			The GeometryInstance3D's max LOD margin.
-			[b]Note:[/b] This property currently has no effect.
+		<member name="visibility_range_begin" type="float" setter="set_visibility_range_begin" getter="get_visibility_range_begin" default="0.0">
+			Starting distance from which the GeometryInstance3D will be visible when [member visibility_range_enabled] is set to true.
 		</member>
-		<member name="lod_min_distance" type="float" setter="set_lod_min_distance" getter="get_lod_min_distance" default="0.0">
-			The GeometryInstance3D's min LOD distance.
-			[b]Note:[/b] This property currently has no effect.
+		<member name="visibility_range_begin_hysteriesis" type="float" setter="set_visibility_range_begin_hysteriesis" getter="get_visibility_range_begin_hysteriesis" default="0.0">
+			Hysteresis margin for the [member visibility_range_begin] threshold.
```

Comment:
Maybe add another sentence about what hysteresis is used for. Something like "hysteresis is the amount of overlap between two visibility ranges allowing both HLODs to draw at the same time to reduce popping in and out"

Reply:
Right, I will add something like that :)